src/ext/components/export-model-input.coffee

```diff
@@ -1,18 +1,32 @@
 H2O.ExportModelInput = (_, _go, modelKey, path, opt={}) ->
   _models = signal []
+  _rawModels = []
```

Comment:
just a minor thing, I think, it would be good, if `_rawModels` will be initialized in the same way as `_models`, so something like `_rawModels = signal []`

Reply:
Ok, fixed.